adhocteam/Head-Start-TTADP
# Remove keyboard shortcuts in calendar

**Description of change**
Remove shortcuts from the calendar

**How to test**
Pull down
Start server
Go to date section in form, you can no longer view shortcuts

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/175

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
